### PR TITLE
Fix the type of vSphere fields in CloudConfig and update auto-generated conversion files.

### PR DESCRIPTION
**Pull request by luomiao**
To make sure the vSphere fields in CloudConfiguration can be successfully serialized
and output into config file.

### CODE REVIEW

**Comment by luomiao** [comment]
This will resolve the problem that cloudconfig fields becoming empty after creat_cluster.

File: pkg/apis/kops/v1alpha1/zz_generated.conversion.go [hunk 1]
```diff
@@ -1250,6 +1256,8 @@ func Convert_kops_KubeSchedulerConfig_To_v1alpha1_KubeSchedulerConfig(in *kops.K
 
 func autoConvert_v1alpha1_KubeletConfigSpec_To_kops_KubeletConfigSpec(in *KubeletConfigSpec, out *kops.KubeletConfigSpec, s conversion.Scope) error {
 	out.APIServers = in.APIServers
+	out.KubeconfigPath = in.KubeconfigPath
```
Comment (SandeepPissay) [inline pkg/apis/kops/v1alpha1/zz_generated.conversion.go:1259]: Is this related to missing vSphere values?

**Review by SandeepPissay — APPROVED** [review]
Looks good to me

File: pkg/apis/kops/v1alpha1/zz_generated.conversion.go (hunk 1 above)
Comment (luomiao) [reply pkg/apis/kops/v1alpha1/zz_generated.conversion.go:1259]: No... I just run "make apimachinery" to update those auto-generated files and these appear automatically.
I guess we have some changes in a previous commit or in upstream repo somebody added something but forgot to update the auto-generated files.
But since these are auto-generated so I think we should leave them untouched.
